lib/genMixin.js

```diff
@@ -6,14 +6,17 @@ const readFile = util.promisify(fs.readFile)
 const writeFile = util.promisify(fs.writeFile)
 
 async function generateMixin(styles, lang, outputFolder) {
+    let templateFile
+    let ext
+    let name
```

Comment:
I wonder if these should have initial values? i.e. `= ""`

Reply:
They could be the default settings. The ones that appear in the else () statement.

Reply:
Yeah that would probably work? Alternatively, make a function:

```js
const getStylesInformation = (lang) => {
     if (lang == "less") {
        return {
            templateFile: "/../lib/template.less.hb"
            ext: "less",
            name: "LESS"
    }
    } else { // if lang == "sass"
        return {
            templateFile: "/../lib/template.scss.hb",
            ext: "scss",
            name: "SASS"
        }
    }
}
```

and return that then use the return from that function?